### PR TITLE
For #7698 - Change active Firebase Test Lab emulator from x86 to ARM in UI jobs

### LINKED ISSUE #7698: Change active Firebase Test Lab emulator from x86 to ARM in UI jobs
Akin to the change in [Fenix](https://github.com/mozilla-mobile/fenix/issues/26792). 

This issue and PR will cover:

* Cleaning up, renaming, and updating our Flank configuration files for the ARM emulator
* Updating ui-test.sh
* Renaming the Jobs in our kind.yml for ARM
* Disabling and or updating some tests that are acting different based on the environment change
* Changing the repo rule from `x86` to `arm` (if there is one)

### PR DESCRIPTION
**Pull request by AaronMT**
For #7698
Fixes #7698
Fixes #7698

### CODE REVIEW

**Comment by AaronMT** [comment]
Focus doesn't look like it has an enforced repository `ui-test-x86` check, so we can go ahead here on Focus. @jcristau not sure if that's something you'd like to stay consistent with Fenix with 🤷🏼 

@sv-ohorvath / @AndiAJ feel free to review test changes
@JohanLorenzo same `automation/taskcluster` changes as Fenix

File: app/src/androidTest/assets/mutedVideoPage.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <button onclick="fullscreen()">Full Screen</button>
     <br><br>
     <video id="mutedVideo" width="420" autoplay muted controls loop>
-        <source src="resources/videoSample.webm" type="video/webm">
+        <source src="resources/clip.mp4" type="video/webm">
```
Comment (sv-ohorvath) [inline app/src/androidTest/assets/mutedVideoPage.html:15]: nit: I forgot to change the type here.

File: app/src/androidTest/assets/videoPage.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <button onclick="fullscreen()">Full Screen</button>
     <br><br>
     <video id="video" width="420" autoplay controls loop>
-        <source src="resources/videoSample.webm" type="video/webm">
+        <source src="resources/clip.mp4" type="video/webm">
```
Comment (sv-ohorvath) [inline app/src/androidTest/assets/videoPage.html:15]: nit: I forgot to change the type here.

File: app/src/androidTest/java/org/mozilla/focus/activity/SettingsGeneralTest.kt [hunk 3]
```diff
@@ -107,6 +108,7 @@ class SettingsGeneralTest {
         }
     }
 
+    @Ignore("Compose scrolling issue: https://github.com/mozilla-mobile/focus-android/issues/7282")
```
Comment (sv-ohorvath) [inline app/src/androidTest/java/org/mozilla/focus/activity/SettingsGeneralTest.kt:111]: I created an issue for the disabled test to keep track of it, if you wish to change the link here: https://github.com/mozilla-mobile/focus-android/issues/7718

**Review by sv-ohorvath — APPROVED** [review]
some small nits, optional, otherwise looks good.

**Review by JohanLorenzo — APPROVED** [review]
The taskcluster changes look good to me!

**Pull request merged**